### ISSUE TITLE
Categorical Cross Entropy , Sparse Categorical Cross Entropy 비교

### ISSUE THREAD

**Issue by CJYjob** [CJYjob/myCodes#1]
Categorical Cross Entropy
Categorical Cross Entropy는 데이터 label이 원-핫 인코딩 방식일 때 사용한다. 우리가 to_categorical을 죽어라 사용하는 것도 이것 때문일 확률이 높다.

Sparse Categorical Cross Entropy
Sparse Categorical Cross Entropy는 반대로 label이 정수일 때 사용한다. 기본적으로 데이터 셋이 제공될 때 label이 정수 형태를 띄고 있는 경우가 많은데 이럴 때 이 loss function을 사용한다.

해당 질문 찾아서, 위 내용으로 수정 필요
Kfold, stratifiedKFold 내용으로 적어놈(클래스간 비율이 깨짐)
